.github/workflows/release.yml

```diff
@@ -7,8 +7,14 @@ on:
         required: true
 
 jobs:
+  go_build_and_test:
```

Comment:
I suggest changing the title of this PR (and later on - squashed commit) from "feat" to "ci" for a clearer changelog